playbooks/postgresql_gcp_staging.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: gcp_postgresql_staging
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../host_vars/gcp_postgres_staging1.yml
+    - ../host_vars/gcp_postgres_staging2.yml
+    - ../group_vars/gcp/gcp_staging_vars.yml
```

Comment:
This is raising the error `ERROR! vars file ../group_vars/gcp/gcp_staging_vars.yml was not found` when I attempt to run this Playbook. Might this not have been included in this pull request?

Reply:
ooof. Sorry about that. Pushed the missing file.